www/base/src/app/settings/service/settings.service.spec.coffee

```diff
@@ -0,0 +1,288 @@
+describe 'settingsService', ->
+    bbSettingsServiceProviderRef = ""
+    beforeEach module "guanlecoja.ui", (bbSettingsServiceProvider) ->
```

Comment:
I think the module should be "app", not 'guanlecoja.ui' I dont know how this can work like this..